docs/src/pages/components/dividers/dividers.md

```diff
@@ -36,5 +36,6 @@ The examples below show two ways of achieving this.
 ## Vertical Dividers
 
 You can also render a divider vertically using the `orientation` prop.
+Note the usage of the `flexItem` prop to accommodate for the flex container.
```

Comment:
```suggestion
Note the use of the `flexItem` prop to accommodate for the flex container.
```

Reply:
Thanks, I take care of it in a follow-up.